spa/src/components/molecules/CookiesBox.vue

```diff
@@ -70,13 +70,64 @@ export default {
   },
   data() {
     return {
-      checked: false,
+      statistical: false,
+      marketing: false,
+      preferencesChosen: false,
     };
   },
+  methods: {
+    confirmAll() {
+      this.statistical = true;
+      this.marketing = true;
+      this.savePreferences();
+    },
+    savePreferences() {
+      const confirmDate = new Date();
+      const options = { statistical: this.statistical, marketing: this.marketing, confirmDate };
+
+      localStorage.setItem('cookieSettings', JSON.stringify(options));
```

Comment:
this should go to vuex store. We use similar context with saved user credentials.